mm2src/common/common.rs

```diff
@@ -1080,8 +1080,18 @@ impl<Id> Default for PagingOptionsEnum<Id> {
 }
 
 #[inline(always)]
+#[cfg(not(feature = "for-tests"))]
 pub fn get_utc_timestamp() -> i64 { Utc::now().timestamp() }
 
+/// get_utc_timestamp for tests allowing to add some bias to 'now'
+#[cfg(feature = "for-tests")]
+pub fn get_utc_timestamp() -> i64 {
+    Utc::now().timestamp()
+        + std::env::var("TEST_TIMESTAMP_OFFSET")
+            .map(|s| s.as_str().parse::<i64>().unwrap_or_default())
+            .unwrap_or_default()
+}
+
```

Comment:
You can use built-in `cfg(test)`

Reply:
I need to call it for other crates and I believe cfg(test) is in effect only for unit tests in same crate
(checked btw - test does not work with cfg(test))

Reply:
there is `run-docker-tests` feature too

Reply:
"for-tests" feature is also used already and looks more general for me (not specific for docker tests)

Reply:
As far as I see you just added `for-tests` feature, idk how it's already there but it's fine, I will make `cfg(test)` working and replace all these test flags in another PR.